sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -198,6 +198,12 @@ object SQLMetrics {
     acc
   }
 
+  def createPartitionMetric(sc: SparkContext, name: String, initValue: Long = -1L): SQLMetric = {
```

Comment:
func name is `createPartitionMetric` but it is using `SIZE_METRIC`. This has the exact code as `createSizeMetric`. So why have a func with different name but same code?

Reply:
I wanted to keep it distinguished from creating size metrics since it's not exactly what we're looking for, but I'm also not sure if `SIZE_METRIC` fits either 😕 do you happen to know the difference between these?

Reply:
Gotcha, thanks! Updated to just use `createSizeMetric` instead